### ISSUE TITLE
Tudou Video cannot be downloaded

### ISSUE THREAD

**Issue by JiaxiangBU** [soimort/you-get#1883]
Please make sure these boxes are checked before submitting your issue – thank you!

- [ ] You can actually watch the video in your browser or mobile application, but not download them with `you-get`.
- [ ] Your `you-get` is up-to-date.
- [ ] I have read <https://github.com/soimort/you-get/wiki/FAQ> and tried to do so.
- [ ] The issue is not yet reported on <https://github.com/soimort/you-get/issues> or  <https://github.com/soimort/you-get/wiki/Known-Bugs>. If so, please add your comments under the existing issue.
- [ ] The issue (or question) is really about `you-get`, not about some other code or project.

Run the command with the `--debug` option, and paste the full output inside the fences:

```
$ you-get --debug 'http://video.tudou.com/v/XMjcxNDE0MDU3Mg==.html?spm=a2h1r.8267915.m1275311.1~3!2~5~5~A'
[DEBUG] get_decoded_html: http://video.tudou.com/v/XMjcxNDE0MDU3Mg==.html?spm=a2h1r.8267915.m1275311.1~3!2~5~5~A
[DEBUG] get_response: http://video.tudou.com/v/XMjcxNDE0MDU3Mg==.html?spm=a2h1r.8267915.m1275311.1~3!2~5~5~A
you-get: version 0.4.715, a tiny downloader that scrapes the web.
you-get: ['http://video.tudou.com/v/XMjcxNDE0MDU3Mg==.html?spm=a2h1r.8267915.m1275311.1~3!2~5~5~A']
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/bin/you-get", line 11, in <module>
    sys.exit(main())
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/__main__.py", line 92, in main
    main(**kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 1429, in main
    script_main('you-get', any_download, any_download_playlist, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 1342, in script_main
    download_main(download, download_playlist, args, playlist, output_dir=output_dir, merge=merge, info_only=info_only, json_output=json_output, caption=caption)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 1158, in download_main
    download(url, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 1422, in any_download
    m.download(url, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/extractors/tudou.py", line 47, in tudou_download
    title = r1(r'\Wkw\s*[:=]\s*[\'\"]([^\n]+?)\'\s*\n', html).replace("\\'", "\'")
AttributeError: 'NoneType' object has no attribute 'replace'```

If there's anything else you would like to say (e.g. in case your issue is not about downloading a specific video; it might as well be a general discussion or proposal for a new feature), fill in the box below; otherwise, you may want to post an emoji or meme instead:

> [WRITE SOMETHING]
> [OR HAVE SOME :icecream:!]

汉语翻译最终日期：2016年02月26日

在提交前，请确保您已经检查了以下内容!

- [ ] 你可以在浏览器或移动端中观看视频，但不能使用`you-get`下载.
- [ ] 您的`you-get`为最新版.
- [ ] 我已经阅读并按 <https://github.com/soimort/you-get/wiki/FAQ> 中的指引进行了操作.
- [ ] 您的问题没有在<https://github.com/soimort/you-get/issues> , <https://github.com/soimort/you-get/wiki/FAQ> 或 <https://github.com/soimort/you-get/wiki/Known-Bugs> 报告，否则请在原有issue下报告.
- [ ] 本问题确实关于`you-get`, 而不是其他项目.

请使用`--debug`运行，并将输出粘贴在下面:

```
[在这里粘贴完整日志]
```

如果您有其他附言，例如问题只在某个视频发生，或者是一般性讨论或者提出新功能，请在下面添加;或者您可以卖个萌:

> [您的内容]
> [舔 :icecream:!]

**Comment by rosynirvana** [comment]
#1708

**Comment by dkl3** [comment]
Additionally, I get this error:

you-get http://www.tudou.com/programs/view/mhWAUb0E0W4/ --debug
[DEBUG] get_content: http://www.tudou.com/programs/view/mhWAUb0E0W4/
[DEBUG] get_response: http://www.tudou.com/outplay/goto/getItemSegs.action?iid=157049561
[DEBUG] get_response: http://ct.v2.tudou.com/f?id=147049649
you-get: version 0.4.715, a tiny downloader that scrapes the web.
you-get: ['http://www.tudou.com/programs/view/mhWAUb0E0W4/']
Traceback (most recent call last):
  File "/usr/local/bin/you-get", line 9, in <module>
    load_entry_point('you-get==0.4.715', 'console_scripts', 'you-get')()
  File "/usr/local/lib/python3.5/dist-packages/you_get-0.4.715-py3.5.egg/you_get/__main__.py", line 92, in main
  File "/usr/local/lib/python3.5/dist-packages/you_get-0.4.715-py3.5.egg/you_get/common.py", line 1441, in main
  File "/usr/local/lib/python3.5/dist-packages/you_get-0.4.715-py3.5.egg/you_get/common.py", line 1352, in script_main
  File "/usr/local/lib/python3.5/dist-packages/you_get-0.4.715-py3.5.egg/you_get/common.py", line 1154, in download_main
  File "/usr/local/lib/python3.5/dist-packages/you_get-0.4.715-py3.5.egg/you_get/common.py", line 1434, in any_download
  File "/usr/local/lib/python3.5/dist-packages/you_get-0.4.715-py3.5.egg/you_get/extractors/tudou.py", line 70, in tudou_download
  File "/usr/local/lib/python3.5/dist-packages/you_get-0.4.715-py3.5.egg/you_get/extractors/tudou.py", line 19, in tudou_download_by_iid
IndexError: list index out of range

**Comment by rosynirvana** [comment]
cannot reproduce
```
you-get -d 'http://www.tudou.com/programs/view/mhWAUb0E0W4/'
[DEBUG] get_decoded_html: http://www.tudou.com/programs/view/mhWAUb0E0W4/
[DEBUG] get_response: http://www.tudou.com/programs/view/mhWAUb0E0W4/
[DEBUG] get_decoded_html: http://www.tudou.com/outplay/goto/getItemSegs.action?iid=157049561
[DEBUG] get_response: http://www.tudou.com/outplay/goto/getItemSegs.action?iid=157049561
[DEBUG] get_html: http://ct.v2.tudou.com/f?id=147049649
[DEBUG] get_response: http://ct.v2.tudou.com/f?id=147049649
Site:       Tudou.com
Title:      20121115 she 超级玛丽欧mario kart 7繁體中文版 ︰ 電視廣告 第一篇
Type:       Flash video (video/x-flv)
Size:       4.02 MiB (4217086 Bytes)

Downloading 20121115 she 超级玛丽欧mario kart 7繁體中文版 ︰ 電視廣告 第一篇.f4v ...
 100% (  4.0/  4.0MB) ├█████████████████████████████████████████┤[1/1]    3 MB/s
```

**Comment by rosynirvana** [comment]
fixed via #1708 and #2015
